### PR TITLE
New Links Added for Icons

### PR DESCRIPTION
**Pull request by Ajay84sia**
New Links Added for Icons

### CODE REVIEW

**Comment by rupali-codes** [comment]
@Ajay84sia can we use these icons in other projects too not just react?

**Comment by Ajay84sia** [comment]
Only for react projects

File: database/frontend/icons.json [hunk 1]
```diff
@@ -75,6 +75,13 @@
   "category": "frontend",
   "subcategory": "icons"
 
+},
+{
+  "name": "React Icons",
+  "description": "Free, high quality, open source icon library for React Projects with over lots of icons.",
```
Comment (CBID2) [inline database/frontend/icons.json:81]:
```suggestion
  "description": "Free, high quality, open source icon library for React Projects.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Ajay84sia! 😄 I made a suggestion for you. I hope you find it helpful! 😄 👍🏾

**Comment by Ajay84sia** [comment]
Hi, @CBID2  I made the changes as you suggested 😄 👍🏾

**Review by CBID2 — APPROVED** [review]
LGTM! 👍🏾

**Comment by rupali-codes** [comment]
> @rupali-codes Only for react projects

thanks

**Pull request merged**